rest-twitter/TweetRetriever_Test.py

```diff
@@ -0,0 +1,11 @@
+#!/bin/python
+
+# Make sure to run with e.g. "env TWITTER_API_KEY=xxx TWITTER_API_KEY_SECRET=xxx python TweetRetriever_Test.py"
+
+from TweetRetriever import TweetRetriever
+
+tr = TweetRetriever()
+
+tweets = tr.get_tweets('cold','39.7392358','-104.990251')
```

Comment:
add a fail scenario

Reply:
can save for later, in order to get full pipeline going right now